### PR TITLE
proofed tezos-client doc and clarified a few bits

### PR DESCRIPTION
**Pull request by narq**

### CODE REVIEW

File: docs/setup/1-tezos-client.md [hunk 1]
```diff
@@ -117,22 +140,27 @@ Error:
 > tutorials to avoid this error. If you use your own local Tezos node you can
 > omit `--wait none`, the error won't happen.
 
-Make a variable for Alice's account address (notice that the address
+Make a variable for Alice's account address  (PHK) (notice that the address
```
Comment (itkach) [inline docs/setup/1-tezos-client.md:143]: It's PKH (public key hash), not PHK. Introduction of this acronym here and without explanation doesn't seem very useful

**Comment by itkach** [comment]
Can you please hard wrap long lines at 80 chars (https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md013---line-length)?

**Comment by itkach** [comment]
>    it's much easier to see and understand the changes in diffs when each sentence is on a single line.  

"One sentence per line" is difficult to consistently implement/enforce, so it's either no wrap or hard wrap, (and sentences can be too long anyway). It's easiest to read diffs of a text that consist of short lines and hard wrapping achieves that.  

>  can reformat my changes to wrap at 80 characters if you like

yes, please

>     * The concept of a wallet address is important and potentially confusing, which is why I introduced the idea of PKH at the first opportunity, said that it was synonymous with the wallet address, and showed where it was stored in the json earlier in the document.  

I am not arguing against mentioning PKH, but all I see is addition of `(PKH)` https://github.com/tqtezos/assets.tqtezos.com/pull/9/commits/14e23ab162eab9d21233fd25a5380ee6f862d377#diff-7e4520ddbb2cf7cb9499b7d5797d25c1R143 without any additional explanation. Leaving it to the reader to figure out from the context is a bit too subtle. What does PKH stand for? Why is it a synonym with "wallet address"? btw, is it a "wallet" or an "address" or a "wallet address" :) ?

File: docs/setup/1-tezos-client.md [hunk 2]
```diff
@@ -75,39 +74,63 @@ Warning:
...
-  we're going to be creating, starting with `tz1...`
-- Activate the account, passing it the path to the `.json` wallet you
-  just downloaded (here, we're calling the account `alice`):
+> With tezos-client installed we can now make a couple of test wallets.
+
+- Go to the [faucet](https://faucet.tzalpha.net/).
+- Complete the captcha and download the wallet in the form of a .json file.
+- The file name is the wallet address (also known as the public key hash, or PKH) with a .json extension.  
+- If you look inside the file you will see a property called pkh which contains the address.
+- The public key hash is used to identify the account on the Tezos block chain and can be though of as an address or account number.
```
Comment (narq) [inline docs/setup/1-tezos-client.md:121]: I briefly introduce the PKH here

**Pull request merged**